diskqueue.go

```diff
@@ -170,6 +174,11 @@ func (d *diskQueue) start() {
 		d.logf(ERROR, "DISKQUEUE(%s) failed to retrieveMetaData - %s", d.name, err)
 	}
 
+	fileNameRegexp, _ = regexp.Compile(`^` + d.name + `.diskqueue.\d\d\d\d\d\d.dat$`)
```

Comment:
there's a chance (though very small) that this gives error. If we want to ensure regexp Compile succeed, either check the error or use [regexp.MustCompile](https://golang.org/pkg/regexp/#MustCompile).

Reply:
Nice catch! Will make the changes :)